pkg/cloudprovider/provider/gce/cloudconfig.go

```diff
@@ -0,0 +1,93 @@
+//
+// Google Cloud Provider for the Machine Controller
+//
+
+package gce
+
+//-----
+// Imports
+//-----
```

Comment:
I'm not a big fan of such comments.
For me this is just noise and decreases readability as `import` keyword is expressive enough for me.
This applies for all "segment" comments in this PR - i'm just commenting on a single bit to have a single conversation.

But this is just my personal opinion.
@alvaroaleman Any opinion on this?

Reply:
Done.